src/js/constants/ServiceSchema.js

```diff
@@ -45,7 +45,7 @@ let SERVICE_SCHEMA = {
           type: 'number',
           default: 1,
           getter: function (service) {
-            return (service.getInstancesCount() || 0);
+            return (service.getInstancesCount() || 0) + '';
```

Comment:
Actually if that's the case I'd rather see an ES6 template string here. ````
```
`${service.getInstancesCount() || 0}`
```